code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -3,6 +3,8 @@
 	name = "\improper Sarathi"
 	id = SPECIES_SARATHI
 	default_color = "00FF00"
+	species_age_min = 17
```

Comment:
```suggestion
	species_age_min = 18
```

Reply:
It is actually the default for other races. This line does nothing tbqh.